packages/menu/test/submenu.test.ts

```diff
@@ -399,6 +399,51 @@ describe('Submenu', () => {
 
         expect(rootItem.open).to.be.false;
     });
+    it('not opens if disabled', async () => {
```

Comment:
Finally got a chance to review these two test failures. It comes from a bug in `Focusable` that is unrelated to your change and that we will be tracking as https://github.com/adobe/spectrum-web-components/issues/2325

To allow this PR to move forward, can you move this test to the end of the test file so that it is not effected by the bug?